src/components/CsvImport/csv.js

```diff
@@ -42,13 +44,22 @@ export default {
         worker: true,
         comments: false,
         step: undefined,
+        chunk: results => {
+          if (Object.keys(parsedData).length === 0 && parsedData.constructor === Object) {
+            parsedData = results
+          } else {
+            parsedData.data = [...parsedData.data, ...results.data]
+            parsedData.errors = [...parsedData.errors, ...results.errors]
+          }
+        },
+        chunkSize: 1024 * 1024 * 10,
```

Comment:
Processing of 10MB chunks takes about 1.5 - 2 seconds. See the profiler screenshot: there are two 10MB chunks, one of them takes 1.62s
![Screenshot from 2021-06-06 23-26-33](https://user-images.githubusercontent.com/24638357/120940743-12e76400-c71f-11eb-9e4b-ccfff814a1e2.png)
That's too long. The page is not responsive. I experimented with a 22MB CSV file with more than a million rows. The best result is when `chunkSize: 1024 * 1024 * 0.7`